### DISCUSSION TITLE
Contributor Call: 3 September 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2613] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 3 september 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/sep-3-2022/4pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Housekeeping:
  - Be mindful of assignments when working on issues/pull requests
  - If you plan to make significant changes to a design, involve the original designer
- Ursula/Unice: Clear up confusion about intent and ownership. (Hopefully) Featuring @nataliasayang & @anna-puk, as well as @BenJamesBen 
- v3 status update

Notes will be added after the call

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @joostdecock 
- @tangerineshark
- @lexander2002 
- @KaerMorhan
- @bobgeorgethe3rd 
- @nataliasayang 
- @anna-puk 
- @BenJamesBen 
- @SeaZeeZee 
- @eriese 
- @nchilada 
- @woutervdub 
- @bbdapalkow

✨ **Joost**
Housekeeping, be mindful of assignments when working on issues/pull requests.
If you’re collaborating on a task with somebody, please pay attention to pull requests and if the task is assigned to anybody. Tasks can be assigned to different people.
If you want to change/update a pattern, please reach out to the creator of the pattern.

✨ **Ursula/Unice issue**
These two designs were merged so they are similar but different. Let’s clear up the confusion.

**Natalia**

> I would add more clear notes on how the pattern is modified to adapt it for different people (for example like adapting Shin for wide hips). I don’t recognize Ursula anymore because of the changes to it so it is harder to maintain it. It would be useful to update the markers on Ursula.

**Anna**

> The changes I made for the markers are possible to be only for Unice, so they can be removed from Ursula.

**_Resolve: Ursula will be brought back a bit to Natalia’s original vision and Unice will be maintained by Anna._**

_Anna’s feedback: how to contact the maintainer of patterns? Right now, there are no contact details for that.
Joost’s, Karen’s and Eriese’s feedback: asking around on Discord right now is the most viable option. Sometimes it is possible to contact the person through GitHub. The lack of contact info is planned to be changed in the future.
Anna’s feedback: maybe it would be useful to put it in the documentation outside of Discord._

✨ **Benjamin&Joost**
Naming the sizing tables for testing the patterns, issue #2706 
The sizes are switched right now. But also further than that, there is a general issue of naming the size tables inside the code. While toying with the pattern sizing, there was a need for a prefix before different sets like “giant” and “doll". Using pronouns doesn’t work so different suggestions are welcome.
There is a need to name two tables of sizes that are based on the measurements of a cis woman and a cis man to test patterns.
Suggestions:
- using human names;
- amab/afab or cis man/cis woman;
- cis female/cis male;
- with breasts/without breasts (or an alternative like full chest/flat chests, flat chest/round chest);
- using something relatively random (like barbie/ken, a/b etc.).

After a conversation, it was settled that **_cis female_** and **_cis male_** will be used.
For the future: is everybody on board with abandoning the current sizing tables?

> For context: sizing tables that are on the website right now (from 3XS to 4XL) are meant here.

Most of the people are on board.
_Natalia’s feedback: the majority of people who just visit the website might be thrown off by the need to put in their own measurements without having size tables to test the patterns with. It might be a problem.
Karen’s feedback: we can add this topic to the agenda of later Contributor Calls so people have more time to ponder this question._

✨ **Joost**
FreeSewing v3 update
All the configuration is at the part-level now. The idea with packs is implemented now, so all of the configurations of the parts are pulled with the parts used from other patterns. Config folder is gone because everything is sorted into parts now.
Also, Joost makes sure to have difference between the core library and how it is used in FreeSewing so people are not tied to how things are done with FreeSewing.

_Wouter’s feedback: what are we doing with existing patterns?
Answer: all the patterns will be adapted for and migrated to v3, but there will still be a v2/2.5 of existing patterns. If there are changes to make into the pattern now (before v3 release), just submit the request.
Same options should better not be entered twice, but later options must be a reference to a previous same option. If options are entered multiple times, the last version will overrule the previous ones._

_Nikhil’s feedback: do you have a lab for v3?
Answer: yes, but not an open one that could be shared on the Internet._

_Zwaluw’s feedback: could I start working on the sleeve packs for Jane?
Answer: yes, you could._

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
